### PR TITLE
Added diagnostics info

### PR DESCRIPTION
**Pull request by otisg**

### CODE REVIEW

File: docs/integration/express.js.md [hunk 1]
```diff
@@ -431,3 +431,13 @@ To add the dependency to your package.json simply use:
 ```
 npm install sematext-express-agent@latest --save
 ```
+
+## Troubleshooting
+
+If you are not seeing some or any Express.js metrics, you can create a "diagnostics dump" and contact us via live chat or email. To create the diagnostics dump just run the following:
+
+```
+sudo spm-express-diagnostics
```
Comment (adnanrahic) [inline docs/integration/express.js.md:440]: sudo spm-express-diagnostics
=>
sudo node ./node_modules/spm-agent-nodejs/bin/spm-nodejs-diagnostics.js

File: docs/integration/express.js.md [hunk 2]
```diff
@@ -431,3 +431,13 @@ To add the dependency to your package.json simply use:
 ```
 npm install sematext-express-agent@latest --save
 ```
+
+## Troubleshooting
+
+If you are not seeing some or any Express.js metrics, you can create a "diagnostics dump" and contact us via live chat or email. To create the diagnostics dump just run the following:
```
Comment (adnanrahic) [inline docs/integration/express.js.md:437]: ...just run the following:

=>

just run the following command in the root directory of your project.

File: docs/integration/node.js.md [hunk 3]
```diff
@@ -338,16 +338,13 @@ The agent will detect you are running PM2 and start collecting metrics automatic
...
 ** Generate diagnostics file for Sematext Support **
 
-If you are not seeing some or any Node.js metrics, you can create a
-"diagnostics dump" and contact us via chat or email. To create the
-diagnostics dump just run the following in your application directory:
+If you are not seeing some or any Node.js metrics, you can create a "diagnostics dump" and contact us via chat or email. To create the diagnostics dump just run the following in your application directory:
+
 ```
-node ./node_modules/spm-agent-nodejs/bin/spm-client-diagnostics.js 
+sudo spm-nodejs-diagnostics
```
Comment (adnanrahic) [inline docs/integration/node.js.md:344]: sudo spm-nodejs-diagnostics
=>
sudo node ./node_modules/spm-agent-nodejs/bin/spm-nodejs-diagnostics.js

File: docs/integration/mongodb.md [hunk 4]
```diff
@@ -21,8 +21,13 @@ Sematext MongoDB monitoring agent is an open-source [mongodb monitoring agent](h
 
 ** Generate diagnostics file for Sematext Support **
 
-If you are not seeing some or all MongoDB metrics, see
-[How do I create the diagnostics package](/monitoring/spm-faq/#how-do-i-create-the-diagnostics-package).
+If you are not seeing some or any MongoDB metrics, you can create a "diagnostics dump" and contact us via live chat or email. To create the diagnostics dump just run the following:
+
+```
+sudo spm-mongodb-diagnostics
```
Comment (adnanrahic) [inline docs/integration/mongodb.md:27]: sudo spm-mongodb-diagnostics
=>
sudo spm-client-diagnostics

File: docs/integration/mongodb.md [hunk 5]
```diff
@@ -21,8 +21,13 @@ Sematext MongoDB monitoring agent is an open-source [mongodb monitoring agent](h
 
 ** Generate diagnostics file for Sematext Support **
 
-If you are not seeing some or all MongoDB metrics, see
-[How do I create the diagnostics package](/monitoring/spm-faq/#how-do-i-create-the-diagnostics-package).
+If you are not seeing some or any MongoDB metrics, you can create a "diagnostics dump" and contact us via live chat or email. To create the diagnostics dump just run the following:
+
+```
+sudo spm-client-diagnostics
```
Comment (otisg) [inline docs/integration/mongodb.md:27]: @adnanrahic this is really the correct name?  No "mongodb" in the name?

Comment (adnanrahic) [reply docs/integration/mongodb.md:27]: Sadly, yes. The Node.js agent had the same name, but it was broken, so while fixing it, I also renamed it into something logical. For this MongoDB agent the script works however. I could rename to the MongoDB diag script but I'd need to release that agent just for that tiny renaming.

I think it's worth it. Give me an hour and I'll make it happen. Then we can keep it like this:

```
sudo spm-mongodb-diagnostics
```

This way we keep a consistent naming across all agents.

**Comment by adnanrahic** [comment]
@otisg Okay this is fixed.
Now the command is:

```
sudo spm-mongodb-diagnostics
```

Version `3.0.4` of the MongoDB agent has this fix.

**Pull request merged**